webpack/.csscomb.json

```diff
@@ -0,0 +1,327 @@
+{
```

Comment:
better defaults for csscomb (groups based on https://github.com/csscomb/csscomb.js/blob/dev/config/csscomb.json)